processing/src/main/java/io/druid/segment/FloatDimensionHandler.java

```diff
@@ -19,17 +19,18 @@
 
 package io.druid.segment;
 
-import io.druid.segment.column.Column;
 import io.druid.segment.column.ColumnCapabilities;
-import io.druid.segment.column.GenericColumn;
-import io.druid.segment.data.Indexed;
+import io.druid.segment.selector.settable.SettableColumnValueSelector;
+import io.druid.segment.selector.settable.SettableFloatColumnValueSelector;
 import io.druid.segment.writeout.SegmentWriteOutMedium;
 
-import java.io.Closeable;
-import java.io.IOException;
+import java.util.Comparator;
 
 public class FloatDimensionHandler implements DimensionHandler<Float, Float, Float>
 {
+  private static Comparator<ColumnValueSelector> FLOAT_COLUMN_COMPARATOR =
+      (s1, s2) -> Float.compare(s1.getFloat(), s2.getFloat());
```

Comment:
please verify nulls are handled properly.

Reply:
Fixed. However, testing this is not the subject of this PR. Please add tests that ensure that columns are merged correctly in the presence of nulls in one of your null-handling PRs.